tests/wallet/nft_wallet/test_nft_offers.py

```diff
@@ -488,7 +472,7 @@ async def test_nft_offer_with_metadata_update(
     updated_nft = coins_maker[0]
     updated_nft_info = match_puzzle(uncurry_puzzle(updated_nft.full_puzzle))
 
-    assert url_to_add in updated_nft_info.also().info["metadata"]  # type: ignore
+    assert url_to_add in disassemble(updated_nft_info.also().info["metadata"])  # type: ignore
```

Comment:
PuzzleDriver actually decodes everything back into Program so I just had to add this: https://github.com/Chia-Network/chia-blockchain/pull/14847/files#diff-c1840ab535355caa37a34dfbb07017e20f08b713378102b8fd4c1a96c574466dR102 to support metadata as Program. Before it was disassembled and assembled back. I assume because of how we use it in other cases.